lib/pure/net.nim

```diff
@@ -825,6 +825,14 @@ when defineSsl:
     else:
       result = getPeerCertificates(socket.sslHandle)
 
+  proc `sessionIdContext=`*(ctx: SslContext, sidCtx: string) =
+      ## Set the session id context, used for resuming handshakes
+      ## Must be unique to application.
+      ## sidCtx must be at most 32 characters in length.
```

Comment:
I think it'd be better if the docs is elaborate more about why this is needed and what is this used for.

Reply:
``` nim
## Sets the session context in which a session can be reused.
## TLS clients might attempt to reuse a session id using the session context, 
## thus it must be set if verifyMode is set to CVerifyPeer or CVerifyPeerUseEnvVars,
## otherwise the connection will fail and SslError will be raised.
## Should be unique per-application to prevent malfunctioning clients.
## sidCtx must be at most 32 characters in length.
```
 How is this?

Reply:
Seems ok, but have a look at what that looks like rendered: `nim doc -d:ssl net.nim`. You need more empty lines in between, otherwise it'll be a huge paragraph and hard to read. I would also suggest using bullet points more here.

Reply:
Might also be a good idea to explain what this would be used for.

Reply:
Ok I updated it, verified the rendered content and it looks fine to me.
```
## Sets the session id context in which a session can be reused.
    ## Used for permitting clients to reuse a session id instead of
    ## doing a new handshake.
    ##
    ## TLS clients might attempt to resume a session using the session id context,
    ## thus it must be set if verifyMode is set to CVerifyPeer or CVerifyPeerUseEnvVars,
    ## otherwise the connection will fail and SslError will be raised if resumption occurs.
    ##
    ## - Only useful if set server-side.
    ## - Should be unique per-application to prevent clients from malfunctioning.
    ## - sidCtx must be at most 32 characters in length.
```

Reply:
It seems you haven't committed these changes?

Reply:
Yes I haven't I just wanted to check if the docs are ok before commiting :)